ui/pages/confirmations/hooks/useConfirmationAlerts.ts

```diff
@@ -1,10 +1,15 @@
 import { useMemo } from 'react';
 import useBlockaidAlerts from './alerts/useBlockaidAlert';
+import useDomainMismatchAlerts from './alerts/useDomainMismatchAlert';
 
 const useConfirmationAlerts = () => {
   const blockaidAlerts = useBlockaidAlerts();
+  const domainMismatchAlerts = useDomainMismatchAlerts();
```

Comment:
Minor, but something I did in the transaction alerts PR to help group these in future, was to create a function for the transaction alerts.

Would it help maintainability if we also made a `useSignatureAlerts` function in this file?

Reply:
I see how you did it in your PR. I like it! updated [07b8a97](https://github.com/MetaMask/metamask-extension/pull/25281/commits/07b8a977e09c97022dd5959c5a83deab421eb366)

I used useMemo in the new function even though it doesn't do much right now. Should be adding another alert hook soon with [Issue - Redesign Signature SIWE: add non-critical alert for account missmatch](https://github.com/MetaMask/metamask-extension/issues/25317)